crates/edit_prediction_button/src/edit_prediction_button.rs

```diff
@@ -1009,3 +1258,730 @@ fn toggle_edit_prediction_mode(fs: Arc<dyn Fs>, mode: EditPredictionsMode, cx: &
         });
     }
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
Leaning towards deleting these tests.  There is just not that much code under tests and it would be very obvious in the UI if it was broken.  Zed doesn't typically have fine grained UI unit tests.

I didn't look all that closely, but it seems like `test_ollama_menu_shows_discovered_models` and `test_ollama_menu_shows_service_models` are testing the same thing.  Those tests also don't appear to be testing code in this module.  The other tests all seem pretty similar and it isn't really testing functionality of this module other than the button not crashing.  Could have one test that does a few things - but it should test outputs of the code in this module.